types/ssz.go

```diff
@@ -82,6 +83,34 @@ func (b SSZTransactions) HashTreeRoot() ([32]byte, error) {
 	return ssz.HashWithDefaultHasher(b)
 }
 
+// SSZWithdrawals --
+type SSZWithdrawals []*capella.Withdrawal
+
+func (b SSZWithdrawals) GetTree() (*ssz.Node, error) {
+	return ssz.ProofTree(b)
+}
+
+func (b SSZWithdrawals) HashTreeRootWith(hh ssz.HashWalker) error {
+	// taken from https://github.com/attestantio/go-eth2-client/blob/bc14358487b6d32cb45feef14b170458abc5d14a/spec/capella/executionpayload_ssz.go#L332-L346
```

Comment:
Why can't this be used directly from the lib?

Reply:
There is no independent implementation for the `Withdrawals HashTreeRootWith` but for the wrapper struct `ExecutionPayload`
see the link attached as a comment